engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html

```diff
@@ -1,12 +1,10 @@
 <span page-title ng-model="contentView">{{ 'Repositories for Content View:' | translate }} {{ contentView.name }}</span>
 
 <header data-block="sub-header">
-  <h3 translate>
-    OSTree Repository Selection
-  </h3>
+  <h3 translate>OSTree Repositories</h3>
 </header>
 
-<nav data-block="navigation">
```

Comment:
The changes to ostree.html come directly from file.html: If it is not in ostree it was not in file. Any fixes I would want to make in both places. Please advise.